### DISCUSSION TITLE
K8S version 1.26 is GA by March 2023, but still it is in preview

### DISCUSSION THREAD

**Discussion by vikree** [Azure/AKS#3590] [General]
According to AKS release calendar, K8S version 1.26 is GA by March 2023, but still it is in preview.  When we will get this version available to implement in production?

**Comment by PixelRobots** [comment]
Hello 1.26 is now GA. You can see the official announcement at https://azure.microsoft.com/en-us/updates/generally-available-kubernetes-126-support-in-aks/
